Source/cmake/WebKitCompilerFlags.cmake

```diff
@@ -460,6 +460,17 @@ int main() {
     unset(CMAKE_REQUIRED_FLAGS)
 endif ()
 
+if (COMPILER_IS_GCC_OR_CLANG)
```

Comment:
Probably OK to run the test regardless of compiler version, right? MSVC doesn't support _Float16 today, but it could in the future.